pkg/tsdb/azuremonitor/azuremonitor.go

```diff
@@ -25,8 +27,12 @@ import (
 	"github.com/grafana/grafana/pkg/tsdb/azuremonitor/types"
 )
 
+var logger = log.New("tsdb.azuremonitor")
+
 func ProvideService(cfg *setting.Cfg, httpClientProvider *httpclient.Provider, tracer tracing.Tracer) *Service {
-	proxy := &httpServiceProxy{}
+	proxy := &httpServiceProxy{
+		logger: logger,
```

Comment:
since you are not using `.FromContext(ctx)` here at any moment, is there any change from previous behavior?

Reply:
No changes. I will revert.